apps/server/test/modules/learnroom/dashboard.e2e-spec.ts

```diff
@@ -9,7 +9,15 @@ import { DashboardResponse } from '@src/modules/learnroom/controller/dto';
 import { JwtAuthGuard } from '@src/modules/authentication/guard/jwt-auth.guard';
 import { DashboardEntity, GridElement, ICurrentUser, User } from '@shared/domain';
 import { IDashboardRepo } from '@shared/repo';
-import { courseFactory, userFactory, createCurrentTestUser } from '@shared/testing';
+import { courseFactory, userFactory, roleFactory } from '@shared/testing';
+
+const mapToCurrentUser = (user: User) =>
```

Comment:
We should move it to testHelpers as mapUserToCurrentUser. 
We should also avoid the user.roles.getItems on this place and pass the roles as second parameter.

Reply:
why pass roles seperately? let's discuss that please.